packages/connect/CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# 9.1.5
+
+-   fix(connect-iframe): not create new timestamp for proxy logs (4919dbcec3)
+-   fix(connect-iframe): user original timestamp for proxy logs (4919dbcec3)
```

Comment:
mmhm I think we had this changelog already agreed on before, from the previous release, didn't we?

Reply:
Sure, I fixed it.